tests/test_custom_models.py

```diff
@@ -444,6 +457,128 @@ def test_repr_lora_conv2d(self):
         self.assertTrue("default" in print_output)
 
 
+class MultipleActiveAdaptersTester(unittest.TestCase):
+    """
+    A test class to test the functionality of multiple active adapters.
+
+    This is not specifically tied to custom models, it's just easy to test here and testing it on all types of models
+    would be overkill.
+    """
+
+    def setUp(self):
+        super().setUp()
+        self.configs = {
```

Comment:
I wonder if we can simplify this: The configs don't really need to live in a `setUp` function, right, they can be a simple constant, similar to the existing `TEST_CASES` one. Then we can get rid of the `TUNERS` constant. Then, if we adopt the suggestion below of explicitly enumerating all config combinations, we can also get rid of the `TARGETS` constant. And then we can also remove `TUNERS_AND_TARGETS` and `_parameterized_custom_name_func`.

I think even though this may be a bit more verbose (or maybe not?), it is more readable and less complex. WDYT?

Reply:
please refer the reply to previous comment

Reply:
addressed in the latest commit